lib/wrappers/genesisProtocol.ts

```diff
@@ -332,13 +333,69 @@ export class GenesisProtocolWrapper extends IntVoteInterfaceWrapper
   }
 
   /**
-   * Return the threshold that is required by a proposal to it shift it into boosted state.
-   * The computation depends on the current number of boosted proposals created for
-   * this voting machine by the DAO or other contract, as well as the GenesisProtocol
-   * parameters thresholdConstA and thresholdConstB.
-   * @param {GetThresholdConfig} options
+   * Return the threshold that is required for a proposal to be shifted into the boosted state.
+   * The computation depends on the current number of boosted proposals that were created for
+   * this voting machine by the scheme at `schemeInfo.address` for the given `creatorAddress`
+   * (`creatorAddress` is the `organizationAddress` given to the `propose` method),
+   * and the GenesisProtocol parameters `thresholdConstA` and `thresholdConstB` which
+   * are registered on behalf of the given scheme with an avatar's Controller,
+   * keyed by the given `creatorAddress`.
+   *
+   * Thus `creatorAddress` must be an avatar address and this function will not work if
+   * you pass for `creatorAddress` any other contract address than an avatar.
+   * Otherwise you must use `getThresholdFromProposal`.
+   *
+   * Note that there will be a separately-scoped threshold for proposals created for this
+   * voting machine by any other scheme than the one given.
+   *
+   * Note that for Arc.js schemes you can just pass an Arc.js contract wrapper for `schemeInfo`.
+   */
+  public async getThresholdForSchemeAndCreator(
+    schemeInfo: ThresholdSchemeInfo,
+    creatorAddress: Address): Promise<BigNumber> {
```

Comment:
is creator the right word here? why not organizationAddress?

Reply:
Because it isn't necessarily an organization.  It can be a contract too.

Reply:
but I realize that for this particular function to work, it must  be an avatar address (as documented), so I'll change that.

Reply:
done